draft-ietf-quic-transport.md

```diff
@@ -843,8 +848,13 @@ which is not a Version Negotiation packet.
 
 A client MUST NOT change the version it uses unless it is in response to a
 Version Negotiation packet from the server.  Once a client receives a packet
-from the server which is not a Version Negotiation packet, it MUST ignore
-Version Negotiation packets on the same connection.
+from the server which is not a Version Negotiation packet, it MUST ignore other
+Version Negotiation packets on the same connection.  Similarly, a client MUST
+ignore a Version Negotiation packet if it has already received and acted on a
+Version Negotiation packet.
+
+A client MUST ignore a Version Negotiation packet if it lists the version it has
+chosen.
```

Comment:
This is difficult to parse. How about "A client MUST ignore a Version Negotiation packet that lists the client's chosen version."